atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/IndexDefinition.java

```diff
@@ -211,18 +173,21 @@ public IndexType getIndexType() {
         return indexType;
     }
 
+    public void validate() {
```

Comment:
I see what you are saying in general (and in this case it doesn't really matter), but that condition is not a early return for the whole method, but just for those statements (in this case those statements are the whole method).  Do you find it clearer to have 
```
if (condition1) {
    return;
}
checks1()
if (condition2) {
    return
}
checks2
```
more concretely would you propose the same change in TableDefinition's validate?

I'll go with the flow on this, but normally I would only do those return early checks if they will always invalidate the whole method call.